models/taskRequests.js

```diff
@@ -476,6 +486,33 @@ const addNewFields = async () => {
   return { documentsModified, totalDocuments };
 };
 
+const addUsersCountAndCreatedAt = async () => {
+  const taskRequestsSnapshots = (await taskRequestsCollection.get()).docs;
+
+  const bulkWriter = firestore.bulkWriter();
```

Comment:
Nit: What is bulkWriter()??

Reply:
https://googleapis.dev/nodejs/firestore/latest/BulkWriter.html